dbt/models/qc/schema.yml

```diff
@@ -868,3 +860,71 @@ models:
               AND procdate NOT BETWEEN '2023-12-19 18:25:28.0' and '2023-12-20 15:07:44.0'
           meta:
             description: asmt_all should be unique by parid, procname, and taxyr
+
+  - name: qc.vw_iasworld_asmt_all_with_prior_year_values
+    description: '{{ doc("view_vw_iasworld_asmt_all_with_prior_year_values") }}'
+
+  - name: qc.vw_report_town_close_neg_asmt_value
+    description: '{{ doc("view_vw_report_town_close_neg_asmt_value") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: Negative ASMT values
+
+  - name: qc.vw_report_town_close_0_land_value
+    description: '{{ doc("view_vw_report_town_close_0_land_value") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: 0 land value
+      export_template: assessed_and_market_values.xlsx
+
+  - name: qc.vw_report_town_close_0_value
+    description: '{{ doc("view_vw_report_town_close_0_value") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: 0 value
+      export_template: assessed_and_market_values.xlsx
+
+  - name: qc.vw_report_town_close_vacant_class_with_bldg_value
+    description: '{{ doc("view_vw_report_town_close_vacant_class_with_bldg_value") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: Vacant class, bldg value
+      export_template: assessed_and_market_values.xlsx
+
+  - name: qc.vw_report_town_close_improved_class_without_bldg_value
+    description: '{{ doc("view_vw_report_town_close_improved_class_without_bldg_value") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: Improved class, no bldg value
+      export_template: assessed_and_market_values.xlsx
+
+  - name: qc.vw_report_town_close_class_does_not_equal_luc
+    description: '{{ doc("view_vw_report_town_close_class_does_not_equal_luc") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: Class does not equal LUC
+
+  - name: qc.vw_report_town_close_500k_increase_1m_decrease
+    description: '{{ doc("view_vw_report_town_close_500k_increase_1m_decrease") }}'
+    config:
+      tags:
+        - qc_report_town_close
+    meta:
+      export_name: 500k increase, 1m decrease
+      export_format:
+        columns:
+          - name: '% Change'
+            index: M
+            horizontal_align: right
```

Comment:
Per the PR body, this is the new syntax for specifying formatting that should be applied to the export workbook at runtime. This is necessary because (as far as I can tell) you can't set horizontal alignment in an Excel template before you populate it with data using openpyxl.